docs/UserGuide.md

```diff
@@ -11,42 +11,61 @@
 Usage: `java -jar RepoSense.jar -config CSV_CONFIG_FILE_PATH [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]`
 1. The report will be generated in the designated OUTPUT_DIRECTORY, or current working directory otherwise.
-1. To visualize the report, open `index.html`.
-1. If the dashboard was not loaded automatically, upload the `archive.zip` (generated in the OUTPUT_DIRECTORY) manually to load the data.
 
-Sample usage:
+Sample usage to generate the report:
 ```
 $ java -jar RepoSense.jar -config CSV_path.csv -output output_path/ -since 01/10/2017 -until 01/11/2017 -formats java adoc js
 ```
-
 Argument List:
 - config : Mandatory. The path to the CSV config file.
 - output : Optional. The path to the dashboard generated. If not provided, it will be generated in the current directory.
 - since : Optional. The start date of analysis. Format: `DD/MM/YYYY`
 - until : Optional. The end date of analysis. Format: `DD/MM/YYYY`
 - formats : Optional. The file formats to analyse. Formats: `alphanumerical file formats`. If not provided, the following file formats will be used. `adoc, cs, css, fxml, gradle, html, java, js, json, jsp, md, py, tag, xml`
 
+## How to View Dashboard
+### With jar
+1. Ensure that you have generated the report.
+1. Execute it on the OS terminal. <br>
+Usage `java -jar RepoSense.jar -view REPORT_DIRECTORY`
+
+```
+java -jar RepoSense.jar -view output_path/reposense-report
+```
+Argument List:
+- view : Mandatory.The server will be started to display the dashboard in the specified directory.
+
+`config` and `view` are mandatory mutually exclusive arguments which means that they cannot be provided together, however, one of them compulsorily has to be present at a given time.
+### Manually
+1. Ensure that you have generated the report.
+1. To visualize the report, open `index.html`.
+1. If the dashboard was not loaded automatically, upload the `archive.zip` (generated in the OUTPUT_DIRECTORY) manually to load the data.
 ```
 Note:
-The contribution calculation is based on the daily commits made within 00:00 to 23:59 in GMT+8.
+The contribution calculation is based on the daily commits made within 00:00 to 23:59 in GMT+8. <br>
+Any other arguments entered with -view will be ignored.
 ```
 
 ### Other option:
 1. Clone this repository (or [download as zip](https://github.com/reposense/RepoSense/archive/master.zip))
 1. Execute the following command on the OS terminal inside the project directory.<br>
-Usage: `gradlew run -Dargs="-config CSV_CONFIG_FILE_PATH [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]"` <br>
+Usage: `gradlew run -Dargs="(-config CSV_path.csv | -view report_path/) [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]"` <br>
 
-Sample usage:
+Sample usage to generate the report:
 ```
 $ gradlew run -Dargs="-config CSV_path.csv -output output_path/ -since 01/10/2017 -until 01/11/2017 -formats java adoc js"
```

Comment:
Should this portion be moved up to the generate report section?

Reply:
No. This one is a part of `other options` for running with gradle where I have included generating dashboard and viewing it together